malaria-ios/ViewControllers/Info Hub and Games/Games Lists/Games/GameViewController.swift

```diff
@@ -56,66 +53,31 @@ class GameViewController: UIViewController {
     presentViewController(alert, animated: true, completion: nil)
   }
   
-  func endGame() {
-    let (title, message) = (GameOverText.title, GameOverText.message)
+  func showEndGameAlert(text: AlertText) {
     
-    let alert = UIAlertController(title: title, message: message, preferredStyle: UIAlertControllerStyle.Alert)
+    let alert = UIAlertController(title: text.title,
+                                  message: text.message,
+                                  preferredStyle: UIAlertControllerStyle.Alert)
     
-    let backAction = UIAlertAction(title: "Back", style: UIAlertActionStyle.Default) {
+    let backAction = UIAlertAction(title: "Back",
+                                   style: UIAlertActionStyle.Default)
+    {
       (action:UIAlertAction!) -> Void in
       
       self.dismissViewControllerAnimated(true, completion: nil)
     }
     
     alert.addAction(backAction)
     
-    let restartGameAction = UIAlertAction(title: "Restart", style: UIAlertActionStyle.Default) {
+    let restartGameAction = UIAlertAction(title: "Restart",
```

Comment:
Move this type of constans to a different extension like so:
https://github.com/systers/malaria-app-ios/blob/develop/malaria-ios/ViewControllers/PlanTrip/PlanTripViewController.swift#L313